mlir/test/lib/Dialect/Vector/TestVectorTransforms.cpp

```diff
@@ -385,16 +385,17 @@ struct TestSinkVectorBroadcast
     registry.insert<memref::MemRefDialect, affine::AffineDialect>();
   }
 
-  StringRef getArgument() const final { return "test-sink-vector-broadcast"; }
+  StringRef getArgument() const final { return "test-vector-reorder-patterns"; }
```

Comment:
You've renamed the argument but the class is still `TestSinkVectorBroadcast`.

Reply:
Good catch, updated!